src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -459,6 +456,36 @@ void RPCServiceImpl::Handle(const impl::MessageToMobile message) {
   }
 }
 
+bool RPCServiceImpl::HandleInvalidVehicleDataRPC(
+    const mobile_apis::FunctionID::eType& api_function_id,
+    const application_manager::commands::MessageSharedPtr message) {
+  switch (api_function_id) {
+    case mobile_apis::FunctionID::GetVehicleDataID: {
+      if (mobile_apis::Result::SUCCESS ==
+          (*message)[strings::msg_params][strings::result_code].asUInt()) {
+        smart_objects::SmartObjectSPtr response =
+            MessageHelper::CreateNegativeResponse(
+                (*message)[strings::params][strings::connection_key].asUInt(),
+                api_function_id,
+                (*message)[strings::params][strings::correlation_id].asUInt(),
+                static_cast<int32_t>(mobile_apis::Result::GENERIC_ERROR));
+
+        SendMessageToMobile(response);
+        return true;
+      }
+    } break;
```

Comment:
@YarikMamykin `break` statement should be inside case

Reply:
@AByzhynar fixed in 73592a6d3e685613557fde5ab85648db5b41a28e